src/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -67,10 +67,10 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*BF16NetworkRestore1.*)",
         R"(.*MobileNet_ssd_with_branching.*)",
 
+        // Gather does not support enforce precision feature.
```

Comment:
Didn't get the comment. Is it regression comparing with previous impl?

Reply:
That was not supported and these are new tests.